e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,181 @@
+import UserAdminPage from 'app/page/admin-user-list-page';
+import { signInWithAccessToken } from 'utils/test-utils';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+import UserProfileInfo from 'app/page/admin-user-profile-info';
+
+describe('Admin', () => {
+  const userEmail = 'admin_test';
```

Comment:
Are you planning to use same tester email name in test and staging? I'm fine with that approach. Just make sure you pre-create the new test users and run tests locally against test and staging first.

Reply:
yes, I'm planning to use the same tester name in test and staging. I've to get access to the admin feature in staging env. Will let you know once it is created and tested locally.